includes/editor-templates/templates.php

```diff
@@ -136,7 +136,7 @@
 
 <script type="text/template" id="tmpl-elementor-template-library-template-remote">
 	<div class="elementor-template-library-template-body">
-		<# if ( 'page' === type ) { #>
+		<# if ( 'page' === type || 'lp' === type ) { #>
```

Comment:
```suggestion
		<# if ( 'page' === type || 'landing-page' === type ) { #>
```

Reply:
It is `lp` because that is the type returned by the library API (@KingYes insisted on lp)

Reply:
Add a comment what is `lp`